parse.go

```diff
@@ -121,8 +121,8 @@ func parseOption(rule string) (string, string, string, error) {
 }
 
 // Parse an IDS rule from the provided string buffer.
-func Parse(buf string) (Rule, error) {
-	rule := Rule{
+func Parse(buf string) (*Rule, error) {
```

Comment:
Why as a pointer? Not saying its wrong, but its worth explaining in the commit message.

Reply:
Sorry, I've just wanted to PR the struct tags. I don't know why the other changes I mede are in the PR.

You can decline all the changes since the one I want to do was the first one `Added json struct tags` 0deb02b.